docs/api.md

```diff
@@ -14,6 +14,7 @@ Install packages.
 * `options.global` - *Boolean* - the packages will be installed globally rather than locally.
 * `options.cwd` - *String* - the directory in which the installation will be performed. By default the `process.cwd()` value is used.
 * `options.quiet` - *Boolean* - `false` by default. No output to the console.
+* `options.cacheTTL` - *Number* - `3.600` seconds  (1 hour) by default. The time during which HTTP requests are cached (except the ones that request tarballs).
```

Comment:
on the other side, 1day cache will help pnpm to cache deps for a whole day and user to enjoy fast install process